spec/javascript/support/puppeteer/consts.js

```diff
@@ -41,3 +41,5 @@ export const DECLINE_TO_STATE = 'Decline to state'
 
 // Change to false to see tests running on browser locally
 export const HEADLESS = true
+// Change to false to fall back to including css
+export const REMOVECSS = true
```

Comment:
can you update to actually make it settable via env var too so we can set it on semaphore?